pkg/operator/controller/externaldns/deployment_test.go

```diff
@@ -57,6 +57,8 @@ var (
 	}
 )
 
+const testSecretHash = "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855"
```

Comment:
this is good - can you create constants  for all duplicated literals in this file

Reply:
Done here v9dd747d85d0c90a7e7c4b1f29746f26be9ee9f2d